src/platform/forms/save-in-progress/reducers.js

```diff
@@ -131,6 +131,7 @@ export function createSaveInProgressInitialState(formConfig, initialState) {
     migrations: formConfig.migrations,
     prefillTransformer: formConfig.prefillTransformer,
     trackingPrefix: formConfig.trackingPrefix,
+    finishLaterLinkText: formConfig.finishLaterLinkText,
```

Comment:
When we originally put pieces of the `formConfig` into the redux store, it was a hack. We never got around to changing that, though.

Ideally, we'd solve this without putting more static data into redux. I think we can do that by adding the `formConfig` to the routes in [`helpers.js`](https://github.com/department-of-veterans-affairs/vets-website/blob/forms-library-book-cv/src/platform/forms/save-in-progress/helpers.js#L34). (See how we do it for special paths like [`form-saved`](https://github.com/department-of-veterans-affairs/vets-website/blob/master/src/platform/forms/save-in-progress/helpers.js#L48-L53) for an example.)

From there, we can use `this.props.route.formConfig` from the `RoutedSavablePage`. (See the [`SaveInProgressErrorPage`](https://github.com/department-of-veterans-affairs/vets-website/blob/master/src/platform/forms/save-in-progress/SaveInProgressErrorPage.jsx#L51) for an example.)

Reply:
@cvalarida I can open a PR to fix this? This was already merged.

Reply:
Sounds great! :raised_hands:

Reply:
I left my review too slowly. :sweat_smile: